### DISCUSSION TITLE
CVE[Common Vulnerability & Exposure] in verdaccio latest

### DISCUSSION THREAD

**Discussion by amzarahamed** [verdaccio/verdaccio#3524] [Q&A] [answered]
We are planning to use verdaccio private npm registry image from docker hub.
But while scanning the verdaccio latest image for CVE's using syft & grype we found several High CVE's.

**CVE image:**
![image](https://user-images.githubusercontent.com/61012389/206719786-8931ce67-376e-4a52-b7cd-3d5a897301ae.png)

**verdaccio Docker image:**
docker pull verdaccio/verdaccio:latest 

We were hoping if someone could provide some valuable insights on this as where to get the latest image with security updates or when we could expect it.

**Comment by juanpicado** [comment, marked as answer]
Those do not seem to be direct dependencies of the project, but I use to rely more on https://snyk.io/advisor/npm-package/verdaccio anyhow, verdaccio updates dependencies often, and should not be a problem using it in a secure way. 

If you have further concerns I'd recommend reading https://verdaccio.org/community/security for any further feedback.
